requirments.txt

```diff
@@ -0,0 +1,74 @@
+asgiref==3.5.2
```

Comment:
are you sure we absolutely need ALL of these software to run a django server that has simple db connection to sqlite3?

Reply:
The first requirement I tried using pip freeze > requirements but that may have saved some extra unnecessary packages.

So I installed pipreqs and that was able to shorten the requirements. It should work.

Reply:
all the pyc. files deleted and pushed the project

Reply:
good. I'll run the project and review the code.